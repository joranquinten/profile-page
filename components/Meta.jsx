import React from 'react';
import Head from 'next/head';
import GlobalStyle from '../styled/Global-style';

import { meta } from '../data/nl';

const Meta = () => (
  <>
    <Head>
      <meta name="viewport" content="width=device-width, initial-scale=1" />
      <meta charSet="utf-8" />
      <link rel="shortcut icon" href="/static/favicon.png" />
      <link rel="stylesheet" type="text/css" href="/static/nprogress.css" />
      <meta name="google-site-verification" content={meta['google-site-verification']} />
      <title>{meta.title}</title>
      <meta name="description" content={meta.description} />

      <link rel="apple-touch-icon" sizes="180x180" href="/static/favicon/apple-touch-icon.png" />
      <link rel="icon" type="image/png" sizes="32x32" href="/static/favicon/favicon-32x32.png" />
      <link rel="icon" type="image/png" sizes="16x16" href="/static/favicon/favicon-16x16.png" />
      <link rel="manifest" href="/static/favicon/site.webmanifest" />
      <link rel="mask-icon" href="/static/favicon/safari-pinned-tab.svg" color="#ed7d3a" />
      <link rel="shortcut icon" href="/static/favicon/favicon.ico" />
      <meta name="msapplication-TileColor" content="#2d89ef" />
      <meta name="msapplication-config" content="/static/favicon/browserconfig.xml" />
      <meta name="theme-color" content="#ffffff" />
    </Head>
    <GlobalStyle />
  </>
);

export default Meta;
