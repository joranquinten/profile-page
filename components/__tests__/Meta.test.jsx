import { shallow } from 'enzyme';

import Meta from '../Meta';

jest.mock('../../data/nl.js');

describe('<Meta />', () => {
  it('should render without crashing', () => {
    expect(shallow(<Meta />)).toMatchSnapshot();
  });
});
