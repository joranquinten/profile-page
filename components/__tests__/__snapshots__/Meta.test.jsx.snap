// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Meta /> should render without crashing 1`] = `
<Fragment>
  <SideEffect(Head)>
    <meta
      content="width=device-width, initial-scale=1"
      name="viewport"
    />
    <meta
      charSet="utf-8"
    />
    <link
      href="/static/icons.png"
      rel="shortcut icon"
    />
    <link
      href="/static/nprogress.css"
      rel="stylesheet"
      type="text/css"
    />
    <meta
      content="metaGoogleSiteVerification"
      name="google-site-verification"
    />
    <title>
      metaTitle
    </title>
    <meta
      content="metaDescription"
      name="description"
    />
    <link
      href="/static/icons/apple-touch-icon.png"
      rel="apple-touch-icon"
      sizes="180x180"
    />
    <link
      href="/static/icons/icons-32x32.png"
      rel="icon"
      sizes="32x32"
      type="image/png"
    />
    <link
      href="/static/icons/icons-16x16.png"
      rel="icon"
      sizes="16x16"
      type="image/png"
    />
    <link
      href="/static/icons/site.webmanifest"
      rel="manifest"
    />
    <link
      color="#ed7d3a"
      href="/static/icons/safari-pinned-tab.svg"
      rel="mask-icon"
    />
    <link
      href="/static/icons/icons.ico"
      rel="shortcut icon"
    />
    <meta
      content="#2d89ef"
      name="msapplication-TileColor"
    />
    <meta
      content="/static/icons/browserconfig.xml"
      name="msapplication-config"
    />
    <meta
      content="#ffffff"
      name="theme-color"
    />
  </SideEffect(Head)>
  <GlobalStyleComponent />
</Fragment>
`;
